CONTRIBUTING.md

```diff
@@ -92,6 +92,13 @@ You need install some dependencies before building a binary.
 
 #### Fedora
 
+  ```shell
+  $ sudo apt-get install -y libsystemd-dev libgpgme-dev libseccomp-dev
+  $ export PKG_CONFIG_PATH="/usr/lib/pkgconfig"
+  ```
+
+#### Debian / Ubuntu
+
```

Comment:
It doesn't seem correct.  The previous command must remain under Fedora, while the new one should be under `Debian / Ubuntu`.